linux_os/guide/system/accounts/accounts-physical/grub2_disable_interactive_boot/rule.yml

```diff
@@ -11,12 +11,20 @@ description: |-
     <tt>yes</tt>, <tt>true</tt>, or <tt>on</tt> value to the
     <tt>systemd.confirm_spawn</tt> kernel argument in <tt>/etc/default/grub</tt>.
     Remove any instance of <pre>systemd.confirm_spawn=(1|yes|true|on)</pre> from
-    the kernel arguments in that file to disable interactive boot. It is also
-    required to change the runtime configuration, run:
+    the kernel arguments in that file to disable interactive boot.
+    Recovery booting must also be disabled. Confirm that
+    <tt>GRUB_DISABLE_RECOVERY=true</tt> is set in  <tt>/etc/default/grub</tt>.
+    It is also required to change the runtime configuration, run:
+    {{% if 'sle' in product %}}
+    <pre>/usr/bin/grub2-editenv - unset systemd.confirm_spawn></pre>
+    <pre>grub2-mkconfig -o /boot/grub2/grub.cfg</pre>
+    {{% else %}}
     <pre>/sbin/grubby --update-kernel=ALL --remove-args="systemd.confirm_spawn"</pre>
+    <pre>/sbin/grubby --update-kernel=ALL --args="GRUB_DISABLE_RECOVERY=true"</pre>
```

Comment:
You probably want to remove this line as well and replace it with grub2-mkconfig.

Reply:
Done and thank you.